cluster/if.go

```diff
@@ -11,5 +12,6 @@ type Node interface {
 	GetPriority() int
 	HasData() bool
 	Post(context.Context, string, string, Traceable) ([]byte, error)
+	PostRaw(ctx context.Context, name, path string, body Traceable) (io.ReadCloser, error)
 	GetName() string
```

Comment:
I'm wondering if it makes sense for both of these methods to exist long-term

Reply:
woops, referring to `Post` and `PostRaw`

Reply:
As a convenience wrapper, probably. `PostRaw` requires explicit management of the body (requires the caller to call `.Close()`). Additionally, as mentioned in https://github.com/grafana/metrictank/pull/1670#discussion_r378162919, the tracing behavior is also different for PostRaw. Both of these *could* be solved by taking a callback instead of returning the body, but callbacks can get confusing.